src/components/NavBar.tsx

```diff
@@ -68,18 +76,17 @@ const NavBar: React.FC = () => {
         >
           <div className="p-4 bg-gray-darkest w-[80vw] min-h-full">
             {nav?.items.map((item) => (
-              <>
+              <div key={item.path}>
                 <Link
-                  key={item.path}
                   to={item.path}
                   className="block font-mono text-2xl p-4 text-white"
                   activeClassName="underline"
                   partiallyActive
                 >
                   {item.title}
                 </Link>
-                {pathname.includes(item.path) && <NavPages />}
-              </>
+                {pathname.includes(item.path) && <NavPages rootItem={item} />}
+              </div>
```

Comment:
Use a [keyed `Fragment`](https://reactjs.org/docs/fragments.html#keyed-fragments) instead of `div`.